.github/workflows/build-test.yaml

```diff
@@ -1,13 +1,13 @@
 name: build-test
 
 on:
-  push:
-    branches:
-      - "**"
+  pull_request_target:
+    types: [opened, synchronize, reopened, labeled]
 
 jobs:
   build-kots:
     runs-on: ubuntu-18.04
+    if: contains(github.event.pull_request.labels.*.name, '@actions/safe-to-test')
```

Comment:
This line should have a comment linking to the document explaining how this can be unsafe

Reply:
It may be possible to add to this conditional "or this pr is coming from dependabot"